example/circuits.js

```diff
@@ -3,7 +3,26 @@
 
 /* eslint-disable  @typescript-eslint/no-unused-vars */
 
+// Basic example: entanglement with measurement
+//export
+const entangle = {
```

Comment:
What was the motivation behind including this sample? Isn't the "entangle" operation already included inside "teleport"?

Also, could we move this example to below "sample" for better ordering?

Reply:
It's just for documentation. `sample` is too complicated for it to be the fist circuit people see. `Entangle` is well understood and a lot simpler to show what the most basic circuit looks like